packages/DropdownMenu/src/components/Content/Content.js

```diff
@@ -0,0 +1,5 @@
+export default function Content() {
```

Comment:
This "component" was required because it needs a `displayName` for `extractChildren` to recognize it.